layouts/almalinux-day-germany-2024/single.html

```diff
@@ -346,7 +359,7 @@ <h4 style="font-size: 23px; letter-spacing: 0.80px; font-weight: 500; margin-bot
             </span>
             <div class="schedule-photo" style="display: flex;">
```

Comment:
<div class="schedule-photo" style="display: flex; flex-wrap: wrap;">

Reply:
Sweet! I'm away from my computer now, but if @mattlasheboro or you want to just suggest the edit I can approve it and then let it re-deploy and maybe that'll do it!